sklearn/kernel_approximation.py

```diff
@@ -778,11 +836,17 @@ def fit(self, X, y=None):
                                         **self._get_kernel_params())
 
         # sqrt of kernel matrix on basis vectors
-        U, S, V = svd(basis_kernel)
+        U, S, V = svd(basis_kernel)  # TODO(kstoneriv3): Why not np.linalg.eigh() ?
```

Comment:
I suppose `eigh` is faster than `svd`. 
https://stackoverflow.com/questions/50358310/how-does-numpy-linalg-eigh-vs-numpy-linalg-svd